GraphColoring/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(GraphColoring)
+
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+find_package(OpenMP)
+find_package(Boost)
+
+FILE(GLOB SRCS
+    "src/graph.cpp"
+    "src/coloring_bench.cpp"
+    "src/perm_repeat_generator.cpp"
+)
+
+set(EXECUTABLE coloring_bench)
+
+add_executable(${EXECUTABLE} ${SRCS})
+target_compile_options(${EXECUTABLE} PUBLIC -Wall -Werror -Wextra -fcoroutines)
+target_link_libraries(${EXECUTABLE} PUBLIC benchmark::benchmark ${Boost_LIBRARIES})
```

Comment:
If you use boost, please provide installation instructions in Readme

Reply:
Same to omp

Reply:
Is it ok to assume that the project will be built in Linux?

Reply:
Sorry for the late response. Let's live with Linux specific build for now, I have no objections.